src/Symfony/Bridge/PhpUnit/DeprecationErrorHandler.php

```diff
@@ -54,9 +54,9 @@ public static function register($mode = 0)
             if (false === $mode) {
                 $mode = getenv('SYMFONY_DEPRECATIONS_HELPER');
             }
-            if (self::MODE_DISABLED !== $mode
-                && self::MODE_WEAK !== $mode
-                && self::MODE_WEAK_VENDORS !== $mode
+            if (DeprecationErrorHandler::MODE_DISABLED !== $mode
```

Comment:
This change isn't needed, is it?

Reply:
That's precisely the line where it fails first :)
https://travis-ci.org/symfony/polyfill/jobs/489417717

Reply:
oh PHP 5.3 :o